habitat_sim/robots/mobile_manipulator.py

```diff
@@ -117,6 +117,9 @@ def __init__(
         self._head_sensor_names = [
             s for s in self._sim._sensors if s.startswith("robot_head_")
         ]
+        self._third_sensor_names = [
+            s for s in self._sim._sensors if s.startswith("robot_third_")
```

Comment:
YIkes, we really need to parameterize these later.

Reply:
I totally agree, this should be a follow up PR, we need a better API for these robot cameras.